### PR TITLE
Rasummer/meta tag updates

### PR DESCRIPTION
**Pull request by rasummer**
@maasg this should fix a bunch of the meta issues. 

Favicons have been fixed. I had copied the Cloudflow favicons to a different Antora skin... my bad. 

You can now define the description and social share image at the page level. Just add the following at the top of the page under the main title that appears on the first line. 

`:description: Description goes here`

`:page-social-share-image: https://cloudflow.io/_/img/cloudflow-logo-social.jpg`

There is no requirement to define the social image. All pages fall back to a generic Cloudflow logo image if you don't define the custom `page-social-share-image`

The rest of the meta tags should build out based on the page title, or they are fixed values and are part of the template.

**PLEASE NOTE**
Certain Antora built-in page attributes didn't seem to show up locally when testing. `page.description` or `page.canonicalUrl` where not showing up, however canonicaUrl seems to be working in production on docs pages, so I'm hoping it's just a local dev issue. I couldn't find any bugs about it. 

Anyway let's see if it works in production. If not then you can fall back to a custom attribute for the page description, which would look list this:

`:page-description: Description goes here`

hope that makes sense. Rich.

### CODE REVIEW

File: preview-src/index.adoc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 = Cloudflow, quickly develop, orchestrate, and operate distributed streaming applications on Kubernetes
+:description: Description goes here
```
Comment (maasg) [inline preview-src/index.adoc]: Should I complete the description that goes here?

File: preview-src/index.adoc [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 :page-layout: home
+:page-social-share-image: https://cloudflow.io/_/img/cloudflow-logo-social.jpg
+
+= This is a heading One
```
Comment (maasg) [inline preview-src/index.adoc]: Is this a test or some kind of marker?

File: preview-src/index.adoc (hunk 1 above)
Comment (rasummer) [reply preview-src/index.adoc]: Yes, you would complete this in the adoc file for the home page. This is just the preview-src file that I work with when skinning.

File: preview-src/index.adoc (hunk 2 above)
Comment (rasummer) [reply preview-src/index.adoc]: Yep, this is me testing that the H1 styles correctly in the skin. Like above this is just a preview-src page that's part of the UI theme.

**Comment by rasummer** [comment]
@maasg I'm online right now and can help test, make updates as needed. Feel free to ping me on Hangouts if needed. Rich

**Pull request merged**